lang.go

```diff
@@ -16,6 +16,7 @@ const (
 	Ceb
 	Ces
 	Cmn
+	Yue
```

Comment:
The languages are in order alphabetically (in English). This line should go above `Zul`.